server/service/src/rnr_form/update.rs

```diff
@@ -209,19 +236,20 @@ fn generate(line_data: Vec<(UpdateRnRFormLine, RnRFormLineRow)>) -> Vec<RnRFormL
                     average_monthly_consumption,
                     stock_out_duration,
                     adjusted_quantity_consumed,
+                    initial_balance, // TODO; snapshot and entered?
```

Comment:
Still a bit unsure about what we want to do if they provide a different initial_balance... we probably want to check whether it matches what we think the initial balance should have been, and create another adjustment as of start of period?

Whether to do this with another snapshot/entered check, or just recalculate on finalise and adjust accordingly?

Reply:
if the process is working then we'll only be taking initial_balance from the previous R&R, and it's only enterable on the first R&R created for the period & program combination. that's creating validation complexity which I don't think we need right now